corehq/apps/accounting/mixins.py

```diff
@@ -8,7 +8,7 @@
     is_subscription_eligible_for_downgrade_process,
 )
 from corehq.apps.accounting.utils.invoicing import (
-    get_oldest_unpaid_invoice_over_threshold,
+    get_oldest_overdue_invoice_over_threshold,
```

Comment:
Confused about unpaid and overdue, what's the difference between this two words? I saw some places you replaced unpaid with overdue, while some other places you replaced overdue with unpaid